tests/integration_tests/functional/test_metrics.py

```diff
@@ -370,3 +373,36 @@ def test_net_metrics(test_microvm_with_api):
         # test that we get metrics while interacting with different interfaces
         net_metrics.validate(test_microvm)
         assert exit_code == 0
+
+
+def test_block_metrics(test_microvm_with_api):
+    """
+    Validate that NetDeviceMetrics doesn't have a breaking change
+    and "net" is aggregate of all "net_*" in the json object.
+    """
+    test_microvm = test_microvm_with_api
+    test_microvm.spawn()
+
+    # Add first scratch block device.
+    fs1 = drive_tools.FilesystemFile(
+        os.path.join(test_microvm.fsfiles, "scratch1"), size=128
+    )
+    test_microvm.add_drive("scratch1", fs1.path)
+
+    # Set up a basic microVM.
+    # (this is the second block device added).
+    test_microvm.basic_config()
+
+    # Add the third block device.
+    fs2 = drive_tools.FilesystemFile(
+        os.path.join(test_microvm.fsfiles, "scratch2"), size=512
+    )
+    test_microvm.add_drive("scratch2", fs2.path)
+
+    num_block_devices = 3
+    block_metrics = FcDeviceMetrics("block", num_block_devices)
+
+    test_microvm.start()
+
+    # check that the started microvm has "block" and "NUM_NET_DEVICES" number of "block_" metrics
```

Comment:
is the mention of net devices still relevant here?

Reply:
Sorry for this, it is a typo, fixed it.